source/Renderer.cpp

```diff
@@ -7,31 +7,35 @@ Renderer::Renderer(Game &game)
   , viewport(game.getWindow().addViewport(camera))
 {
   scenemgr->setShadowTechnique(Ogre::SHADOWTYPE_STENCIL_MODULATIVE);
+
+  Ogre::OverlaySystem* pOverlaySystem = new Ogre::OverlaySystem();
```

Comment:
inlining isn't always using a keyword :P
right now you have something like:
```cpp
A a(...)
b(a)
```
inlining a:
```cpp
b(A(...))
```